packages/components/addon/components/hds/tabs/tab.hbs

```diff
@@ -0,0 +1,19 @@
+{{! template-lint-disable require-context-role }}
+{{! template-lint-disable no-invalid-role }}
+
+<li class={{this.classNames}} role="presentation">
+  <a
+    role="tab"
+    class="hds-typography-body-200 hds-font-weight-medium"
+    href={{concat "#" this.panelId}}
```

Comment:
I'm using JavaScript to enhance the native functionality essentially in this case. I don't think I should be using anchor tags with empty hrefs even if the JavaScript prevents them from being navigated to.

@MelSumner @alex-ju opinions?

Reply:
In an implementation where the non-active panels are hidden with CSS (as it is Heydon's example you referenced), using an anchor that points to a section makes total sense, because if you strip the page of JS/CSS (or you use progressive enhancement) you get a set of links pointing to different sections of a page. Now, in Ember this is not the case, the non-active panels are not hidden from the page, they're not part of the DOM at all until a tab is pressed (when they get injected). For this reason, I tend to think that a `<button>` element would be a better fit for our tabs.

Reply:
Dismiss my comment above, we are actually showing and hiding the panels 🤦  – in which case we can eventually choose to not suppress the segment from being added to the URL and make sure a direct link would work by activating the tag that matches the segment (maybe not a feature for the basic functionality, but something to consider).

Reply:
@MelSumner Thanks. For now I'll just remove the preventDefault

Reply:
@KristinLBradley @MelSumner please make a note to validate this assumption with the Engineering ambassadors (as a consumer, I would prefer to have control of the routes/URLs in my application and this may impact consumer-side logic in some cases).